elfi/methods/mcmc.py

```diff
@@ -374,7 +374,7 @@ def _build_tree_nuts(params, momentum, log_slicevar, step, depth, log_joint0, ta
             mh_ratio, n_steps, is_div, is_out
 
 
-def metropolis(n_samples, params0, target, sigma_proposals, seed=0):
+def metropolis(n_samples, params0, target, sigma_proposals, warmup=1, seed=0):
```

Comment:
Why is the default `warmup=1`?

Reply:
It referred to the initial parameter `params0` (which wouldn't perhaps be counted as warmup)